db/seeds/transportation_offices.json

```diff
@@ -5364,8 +5364,8 @@
       "postal_code": "77034",
       "country_name": "United States",
       "country_code": "US",
-      "latitude": 29.837692,
-      "longitude": -95351038.0
+      "latitude": 29.609559,
```

Comment:
updated both of these based on what i got from entering the address into google maps. the original coordinates were kinda off.